tests/zfs-tests/tests/functional/cli_root/zpool_expand/zpool_expand_001_pos.ksh

```diff
@@ -27,6 +27,7 @@
 
```

Comment:
@don-brady, I revised my commit to not alter test 001, except that I removed unnecessary use of variables EX_{1,3}GB.